spinalcordtoolbox/process_seg.py

```diff
@@ -39,12 +42,30 @@ def compute_shape(segmentation, angle_correction=True, param_centerline=None, ve
                      ]
 
     im_seg = Image(segmentation).change_orientation('RPI')
-
     # Getting image dimensions. x, y and z respectively correspond to RL, PA and IS.
     nx, ny, nz, nt, px, py, pz, pt = im_seg.dim
+    pr = min([px, py])
+    # Resample to isotropic resolution in the axial plane. Use the minimum pixel dimension as target dimension.
+    # Note: need to convert to nibabel, then nipy (input type of resample_nipy)
+    im_seg_nibabel = nibabel.nifti1.Nifti1Image(im_seg.data, im_seg.hdr.get_best_affine())
+    im_segr_nipy = resample_nipy(
+        nifti2nipy(im_seg_nibabel), new_size=[pr, pr, pz], new_size_type='mm', interpolation='linear')
+    # Convert back to Image type
+    im_segr_nibabel = nipy2nifti(im_segr_nipy)
+    im_segr = Image(im_segr_nibabel.get_data(), hdr=im_segr_nibabel.header, orientation='RPI', dim=im_segr_nibabel.header.get_data_shape())
+
+
+    # img = msct_image.Image(i.get_data(), hdr=i.header,
```

Comment:
Do we need to leave this code commented out here?

Reply:
no, we can remove